channeldb/graph.go

```diff
@@ -952,6 +966,8 @@ func (c *ChannelGraph) UpdateChannelEdge(edge *ChannelEdgeInfo) error {
 			return ErrEdgeNotFound
 		}
 
+		c.graphCache.UpdateChannel(edge)
+
```

Comment:
It looks like this is only called by `AddProof`, so just updating a field withing the edge. So we might not even need the update to the cache at this point. But I'd like to keep it just in case we ever add something later that we _do_ want to track in the cache. But we certainly shouldn't wipe anything here.